streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/service/StreamCatalogService.java

```diff
@@ -2341,4 +2343,55 @@ private String getTopologyComponentBundleJarName (TopologyComponentBundle topolo
     private Collection<File> listFiles(List<QueryParam> queryParams) {
         return dao.find(File.NAME_SPACE, queryParams);
     }
+
+    public Collection<TopologyTestRunHistory> listTopologyTestRunHistory(Long topologyId) {
```

Comment:
Does it make sense to add version id as well to some of these methods that have topology id as argument?